learnMarkdown.md

```diff
@@ -0,0 +1,20 @@
+# Big heading
+## Slightly Smaller heading
+
+* Bullet 1
+* Bullet 2
+
+**bold**
+
+*italics*
+
+~~strikethrough~~
+
+__bold__
+-italics_
+
+[I'm an inline style link] https://twitter.com/cgrayy
```

Comment:
Actually, the link should be encapsulated in parens.
e.g `[I'm an inline style link](https://twitter.com/cgrayy)`